handlers/addNewProduct.js

```diff
@@ -0,0 +1,84 @@
+const mysql = require('mysql')
+const { v4: uuidv4 } = require('uuid')
+require('dotenv').config()
+
+module.exports.createProduct = async (event, context) => {
+  console.log('Created database client!')
+  const client = mysql.createConnection({

```

Comment:
It is makes sense to create connection outside of handler at the root of module.